ElectronicObserver/Window/Tools/SortieRecordViewer/Sortie/Battle/Phase/PhaseNightInitial.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using ElectronicObserver.Data;
+using ElectronicObserver.KancolleApi.Types.Interfaces;
+using ElectronicObserverTypes;
+using ElectronicObserverTypes.Data;
+using ElectronicObserverTypes.Extensions;
+
+namespace ElectronicObserver.Window.Tools.SortieRecordViewer.Sortie.Battle.Phase;
+
+public class PhaseNightInitial : PhaseBase
+{
+	public override string Title => "夜戦開始";
```

Comment:
Have tl for this
Maybe just add a todo ?

Reply:
Added translation.